elementary/config/config.py

```diff
@@ -13,18 +13,20 @@ class Config:
 
     DEFAULT_CONFIG_DIR = str(Path.home() / '.edr')
     DEFAULT_PROFILES_DIR = str(Path.home() / '.dbt')
+    DEFAULT_TARGET_DIR = 'output'
```

Comment:
I believe this should be called `edr_output`.
Given that `edr` can be run from every directory,
it might be confusing to just have the `output` directory.

Reply:
Agree, you're right